### DISCUSSION TITLE
Can I move my wallet to my new computer?

### DISCUSSION THREAD

**Discussion by felik-panda** [DeFiCh/app#795] [Q&A] [answered]
Hi, I'm Felik. I just bought new Computer, can i move my wallet to my new computer? Could you help me step by step to move my wallet? Thanksss...

**Comment by izzycsy** [comment, marked as answer]
Hi @felik-panda , yes you can. 
1. Make a copy of your wallet.dat (this is a hidden folder) 

Windows: 
`C:\Users\<username>\AppData\Roaming\DeFi Blockchain\wallets`

Mac — Go to Folder: 
`/Users/<username>/Library/Application Support/Defi/wallets`

<img width="822" alt="wallet dat location" src="https://user-images.githubusercontent.com/65014479/116521653-d39b4b80-a906-11eb-861a-3ea03a986121.png">

2. Transfer `wallet.dat` to your new PC

> You can save it in Documents folder so it's easier to access it when restoring wallet

3. Ensure you've at least 130GB storage and install [DeFi Wallet ](https://defichain.com/downloads/)
4. Restore Wallet via backup (wallet.dat)

<img width="1021" alt="Restore wallet from backup" src="https://user-images.githubusercontent.com/65014479/116521765-f75e9180-a906-11eb-8842-492b97a4f699.png">
